azurelinuxagent/common/event.py

```diff
@@ -320,7 +320,7 @@ def report_dropped_events_error(count, errors, operation_name):
     def _update_errors_and_get_count(error_count, errors, error):
         error_count += 1
         if len(errors) < CollectOrReportEventDebugInfo.__MAX_ERRORS_TO_REPORT:
-            errors.add("{0}: {1}".format(ustr(error), traceback.format_exc()))
+            errors.add("{0}: {1}".format(ustr(error), textutil.format_exception()))
```

Comment:
I think textutil.format_exception should  take the exception as parameter, otherwise we have an (implicit) requirement that is must be called within an exception handler and in some case, as this one, that is not clear/enforced in the code.